### PR TITLE
add s3cmd installation step in Dockerfile (Jewel, Ubuntu 16.04)

### LINKED ISSUE #611: ceph-demo crashes when specify CEPH_DEMO_BUCKET (Jewel, Ubuntu 16.04)
Problem is caused by missing s3cmd in docker image (**ceph/demo:tag-build-master-jewel-ubuntu-16.04**):
`/entrypoint.sh: line 349: s3cmd: command not found`

### PR DESCRIPTION
**Pull request by xakep139**
This fixes #611

### CODE REVIEW

File: ceph-releases/jewel/ubuntu/16.04/demo/Dockerfile [hunk 1]
```diff
@@ -7,6 +7,13 @@
 FROM ceph/base:tag-build-master-jewel-ubuntu-16.04
 MAINTAINER Sébastien Han "seb@redhat.com"
 
+# Install s3cmd to support creation of buckets
+RUN ["apt-get", "update", "-y"]
+RUN ["apt-get", "install", "-y", "python-dateutil", "python-magic"]
+
+RUN ["wget", "http://mirrors.kernel.org/ubuntu/pool/universe/s/s3cmd/s3cmd_1.6.1-1_all.deb"]
+RUN ["dpkg", "-i", "s3cmd_1.6.1-1_all.deb"]
```
Comment (leseb) [inline ceph-releases/jewel/ubuntu/16.04/demo/Dockerfile]: Isn't `s3cmd` already available as part of Ubuntu 16.04?

Comment (xakep139) [inline ceph-releases/jewel/ubuntu/16.04/demo/Dockerfile]: When specifying `CEPH_DEMO_BUCKET` environment variable on Docker image `ceph/demo:tag-build-master-jewel-ubuntu-16.04` I get an error:
```
2017-04-24 09:41:23  /entrypoint.sh: Creating bucket...
2017-04-24 09:41:23  /entrypoint.sh: Transforming your bucket name to uppercase.
2017-04-24 09:41:23  /entrypoint.sh: It appears there is a bug in s3cmd 1.6.1 with lowercase bucket names.
/entrypoint.sh: line 349: s3cmd: command not found
```

Comment (leseb) [inline ceph-releases/jewel/ubuntu/16.04/demo/Dockerfile]: I meant there is no need to fetch the `.deb`, can't we just install s3cmd from the Ubuntu package source?

Comment (xakep139) [inline ceph-releases/jewel/ubuntu/16.04/demo/Dockerfile]: Yeah, I've got your idea, thanks!

**Comment by leseb** [comment]
@xakep139 please squash your commits into a single one.

**Comment by xakep139** [comment]
You can easily squash commits using GitHub's merge options: https://help.github.com/articles/about-pull-request-merges/#squash-and-merge-your-pull-request-commits

select-squash-and-merge-from-drop-down-menu

**Comment by leseb** [comment]
@xakep139 that's right, never used that option before.

**Pull request merged**